src/test/java/api/Data.java

```diff
@@ -0,0 +1,281 @@
+package api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(as = Data.class)
+public class Data {
+    @JsonProperty("Ref")
+    public String ref;
+    @JsonProperty("SettlementType")
+    public String settlementType;
+    @JsonProperty("Latitude")
+    public String latitude;
+    @JsonProperty("Longitude")
+    public String longitude;
+    @JsonProperty("Description")
+    public String description;
+    @JsonProperty("DescriptionRu")
+    public String descriptionRu;
+    @JsonProperty("DescriptionTranslit")
+    public String descriptionTranslit;
+    @JsonProperty("SettlementTypeDescription")
+    public String settlementTypeDescription;
+    @JsonProperty("SettlementTypeDescriptionRu")
+    public String settlementTypeDescriptionRu;
+    @JsonProperty("SettlementTypeDescriptionTranslit")
+    public String settlementTypeDescriptionTranslit;
+    @JsonProperty("Region")
+    public String region;
+    @JsonProperty("RegionsDescription")
+    public String regionsDescription;
+    @JsonProperty("RegionsDescriptionRu")
+    public String regionsDescriptionRu;
+    @JsonProperty("RegionsDescriptionTranslit")
+    public String regionsDescriptionTranslit;
+    @JsonProperty("Area")
+    public String area;
+    @JsonProperty("AreaDescription")
+    public String areaDescription;
+    @JsonProperty("AreaDescriptionRu")
+    public String areaDescriptionRu;
+    @JsonProperty("AreaDescriptionTranslit")
+    public String areaDescriptionTranslit;
+    @JsonProperty("Index1")
+    public String index1;
+    @JsonProperty("Index2")
+    public String index2;
+    @JsonProperty("IndexCOATSU1")
+    public String indexCOATSU1;
+    @JsonProperty("Delivery1")
+    public String delivery1;
+    @JsonProperty("Delivery2")
+    public String delivery2;
+    @JsonProperty("Delivery3")
+    public String delivery3;
+    @JsonProperty("Delivery4")
+    public String delivery4;
+    @JsonProperty("Delivery5")
+    public String delivery5;
+    @JsonProperty("Delivery6")
+    public String delivery6;
+    @JsonProperty("Delivery7")
+    public String delivery7;
+    @JsonProperty("SpecialCashCheck")
+    public int specialCashCheck;
+    @JsonProperty("Warehouse")
+    public String warehouse;
+
+    @JsonCreator
+    public Data(@JsonProperty("ref") String ref, @JsonProperty("settlementType")String settlementType,
```

Comment:
(@JsonProperty("**R**ef") instead of ref and the same for others fields
The error ClassCastException happens due to impossibility to parse pojo
Java is the case sensitive language